ui/pages/swaps/prepare-swap-page/not-enough-balance.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  SEVERITIES,
+  TextVariant,
+  Size,
+} from '../../../helpers/constants/design-system';
+import {
+  BannerAlert,
+  ButtonLink,
+  Text,
+} from '../../../components/component-library';
+
+interface Props {
+  title: string;
+  actionableBalanceErrorMessage: string;
+  needsMoreGas: boolean;
+  openBuyCryptoInPdapp: () => void;
+  needsMoreGasText: string;
+}
+
+export default function NotEnoughBalance({
+  title,
+  actionableBalanceErrorMessage,
+  needsMoreGas,
+  openBuyCryptoInPdapp,
+  needsMoreGasText,
+}: Props) {
+  return (
+    <BannerAlert severity={SEVERITIES.INFO} title={title} className="">
+      <Text
+        variant={TextVariant.bodyMd}
+        as="h6"
+        data-testid="mm-banner-alert-notification-text"
+      >
+        {actionableBalanceErrorMessage}
+
+        {needsMoreGas && (
+          <ButtonLink
+            onClick={() => openBuyCryptoInPdapp()}
+            size={Size.inherit}
+            className=""
+            danger={false}
+            disabled={false}
+            style={{ verticalAlign: 'baseline' }}
```

Comment:
I think we can avoid this inline style by adding style to the class name. wdyt?

Reply:
For this one I can make a new `className` with the inline styles

Reply:
Resolved in 9778cd2